assets/src/apps/authoring/store/groups/layouts/deck/actions/addSequenceItem.ts

```diff
@@ -0,0 +1,73 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import {
+  findInHierarchy,
+  flattenHierarchy,
+  getHierarchy,
+} from '../../../../../../delivery/store/features/groups/actions/sequence';
+import { GroupsSlice, upsertGroup } from '../../../../../../delivery/store/features/groups/slice';
+
+export const addSequenceItem = createAsyncThunk(
+  `${GroupsSlice}/addSequenceItem`,
+  async (payload: any, { dispatch, getState }) => {
```

Comment:
from the surround code
```suggestion
  async (payload: {
      sequence?: SequenceEntry<SequenceEntryChild>[],
      group?: any,
      item?: SequenceHierarchyItem<SequenceEntryChild>,
      parentId?: string,
      siblingId?: string,
    }, { dispatch }) => {
```
looks about right, or looser version could be

```suggestion
  async (payload: {
      sequence?: any[],
      group?: any,
      item?: any,
      parentId?: string,
      siblingId?: string,
    }, { dispatch }) => {
```